### PR TITLE
[HttpKernel] Add `#[WithLogLevel]` attribute

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Closes https://github.com/symfony/symfony-docs/issues/17658

CI will be solved by https://github.com/symfony/symfony-docs/pull/17668

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -3611,6 +3611,25 @@ use the configuration of the first exception that matches ``instanceof``:
                     log_level: 'debug'
                     status_code: 422
 
+It is also possible to map a log level on a custom exception class using the
+``#[WithLogLevel]`` attribute:
+
+.. code-block:: php
```
Comment (OskarStark) [inline reference/configuration/framework.rst]:
```suggestion
``#[WithLogLevel]`` attribute::
```

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Rebase needed

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -3611,6 +3611,25 @@ use the configuration of the first exception that matches ``instanceof``:
                     log_level: 'debug'
                     status_code: 422
 
+It is also possible to map a log level on a custom exception class using the
+``#[WithLogLevel]`` attribute::
+
+    <?php
```
Comment (OskarStark) [inline reference/configuration/framework.rst]:
```suggestion
```

Comment (alexandre-daubois) [reply reference/configuration/framework.rst]: Updated, thanks for the review!

**Comment by javiereguiluz** [comment]
Thanks Alexandre!

**Pull request merged**